src/components/ReportDetailed/index.js

```diff
@@ -1,109 +1,136 @@
-// @flow
+  // @flow
```

Comment:
You accidentally indented the whole file making it really hard to see your actual changes. If you indent it back we can see what you changed.